src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncLocalsTests.cs

```diff
@@ -1273,5 +1273,263 @@ public class C
                 }, module.GetFieldNames("C.<F>d__0"));
             });
         }
+
+        [WorkItem(9775, "https://github.com/dotnet/roslyn/issues/9775")]
+        [Fact]
+        public void Fixed_Debug()
```

Comment:
>public void Fixed_Debug() [](start = 8, length = 25)

Is it possible to get in a situation when a pinned local has to be spilled? Do we have any tests for scenarios like this?